joomla/joomla-cms
# Normalize #__ucm_content table across 3 db systems and add default values

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/13510 and probably other issues relating ucm_content table in more strict db systems (which in joomla 4.0 mysql will be also).

### Summary of Changes

Normalize the `#__ucm_content` table across db systems.
Only tested in mysql.
@waader please check mssql
@alikon please also check postgresql and mssql.

### Testing Instructions

1. SQL code review 
2. Test update: Apply patch, manually run the sql query for your Db system, confirm db table, confirm all fine
3. Test install: apply patch, delete configuration.php do a normal install, confirm db table, confirm all fine

### Documentation Changes Required

None.
